reloj-checador/app/helpers/application_helper.rb

```diff
@@ -1,2 +1,16 @@
 module ApplicationHelper
+  def flash_msg
+    flash.each_with_object([]) do |(type, message), flash_messages|
+      # p "Entre aqui #{message} #{type}"
```

Comment:
Look how solidus builds the flash messages: https://github.com/solidusio/solidus/blob/a17109b07a817ae34bc61491c3ed1ea824ee32c5/core/app/helpers/spree/base_helper.rb#L67

Reply:
Comment taken and modifying to changed for toastr used instead Bootstrap 4 alerts.